index.js

```diff
@@ -17,22 +17,22 @@ module.exports = {
     }
   },
 
-  included() {
+  included(app) {
     this._super.included.apply(this, arguments);
 
-    let app;
-
-    // If the addon has the _findHost() method (in ember-cli >= 2.7.0), we'll just
-    // use that.
-    if (typeof this._findHost === 'function') {
-      app = this._findHost();
-    } else {
-      // Otherwise, we'll use this implementation borrowed from the _findHost()
-      // method in ember-cli.
-      let current = this;
-      do {
-        app = current.app || app;
-      } while (current.parent.parent && (current = current.parent));
+    if (typeof app.import !== 'function') {
```

Comment:
Can you add a comment above that explains why we're doing a conditional here so that future users can better understand.

Reply:
Updated, thank you!